src/apps/intermediate-list/fee-list/fee-list.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from "react";
+import { FC, useEffect, useState } from "react";
+
+import { useGetFeesV2 } from "../../../core/fbs/fbs";
+import { FeeV2 } from "../../../core/fbs/model";
+import { useText } from "../../../core/utils/text";
+import FeeListItem from "../fee-list-item/fee-list.item";
+import TotalPaymentPay from "../stackable-fees/total-payment-pay";
+import {
+  getFeesPostPaymentChangeDate,
+  getFeesPrePaymentChangeDate
+} from "../utils/intermediate-list-helper";
+
+const FeeList: FC = () => {
+  const t = useText();
+  const { data: fbsFees } = useGetFeesV2<FeeV2>();
+  const [itemsPrePaymentChange, setItemsPrePaymentChange] = useState<
+    FeeV2[] | boolean
+  >(false);
+  const [totalFeePrePaymentChange, setTotalFeePrePaymentChange] =
+    useState<number>(0);
+  const [itemsPostPaymentChange, setItemsPostPaymentChange] = useState<
+    FeeV2[] | boolean
+  >(false);
+  const [totalFeePostPaymentChange, setTotalFeePostPaymentChange] =
+    useState<number>(0);
+
+  useEffect(() => {
+    if (fbsFees) {
+      const feesPrePaymentChange = getFeesPrePaymentChangeDate(fbsFees).length;
+      if (feesPrePaymentChange > 0) {
+        setItemsPrePaymentChange(getFeesPrePaymentChangeDate(fbsFees));
+      }
+      const feesPostPaymentChange = getFeesPrePaymentChangeDate(fbsFees).length;
+      if (feesPostPaymentChange > 0) {
+        setItemsPostPaymentChange(getFeesPostPaymentChangeDate(fbsFees));
+      }
+    }
+  }, [fbsFees]);
+
+  useEffect(() => {
+    let totalFee = 0;
+    Object.values(itemsPrePaymentChange).forEach((item) => {
+      totalFee += item.amount;
+    });
+    setTotalFeePrePaymentChange(totalFee);
+  }, [itemsPrePaymentChange]);
+
+  useEffect(() => {
+    let totalTally = 0;
+    Object.values(itemsPostPaymentChange).forEach((item) => {
+      totalTally += item.amount;
+    });
+    setTotalFeePostPaymentChange(totalTally);
+  }, [itemsPostPaymentChange]);
+
+  return (
+    <div>
+      {!itemsPrePaymentChange && !itemsPostPaymentChange && (
+        <>
+          <p style={{ textTransform: "uppercase" }}>
+            {t("unpaidFeesText")} <sup>0</sup>
```

Comment:
is this like this: 

https://github.com/danskernesdigitalebibliotek/dpl-react/blob/b6b50004fa3e5114b3db7c4598dd03e4dbd7a9c6/src/apps/loan-list/list/ListHeader.tsx#L14

Reply:
<img width="176" alt="image" src="https://user-images.githubusercontent.com/15377965/209084256-64ce38e5-02ec-4a5e-a8e3-70e7b6f69ac9.png">

altså den?